src/App.jsx

```diff
@@ -14,70 +16,59 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  const selectGood = (good) => {
+    setSelectedGood(good);
+  };
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {selectedGood ? `${selectedGood} is selected` : 'No goods selected'}
+        {selectedGood && (
+          <button
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+            onClick={() => selectGood('')}
+          />
+        )}
```

Comment:
Instead of two conditions you can use one with react fragment
```suggestion
        {selectedGood 
          ? <>
              `${selectedGood} is selected`
              <button
                 data-cy="ClearButton"
                 type="button"
                 className="delete ml-3"
                 onClick={() => selectGood('')}
                 />
            </> 
          : 'No goods selected'}
```